types/blueimp-load-image/blueimp-load-image-tests.ts

```diff
@@ -5,78 +5,99 @@ import loadImage = require('blueimp-load-image');
-  'AAMBAQEBAQEBAQEAAAAAAAABAgMEBQYHCAkKC//EALURAAIBAgQEAwQHBQQEAAEC' +
-  'dwABAgMRBAUhMQYSQVEHYXETIjKBCBRCkaGxwQkjM1LwFWJy0QoWJDThJfEXGBka' +
-  'JicoKSo1Njc4OTpDREVGR0hJSlNUVVZXWFlaY2RlZmdoaWpzdHV2d3h5eoKDhIWG' +
-  'h4iJipKTlJWWl5iZmqKjpKWmp6ipqrKztLW2t7i5usLDxMXGx8jJytLT1NXW19jZ' +
-  '2uLj5OXm5+jp6vLz9PX29/j5+v/aAAwDAQACEQMRAD8A/v4ooooA/9k=';
+    '/9j/4AAQSkZJRgABAQEAYABgAAD/4QAiRXhpZgAASUkqAAgAAAABABIBAwABAAAA' +
+    'BgASAAAAAAD/7QAsUGhvdG9zaG9wIDMuMAA4QklNBAQAAAAAAA8cAgUACm9iamVj' +
+    'dG5hbWUA/9sAQwABAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEB' +
+    'AQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEB/9sAQwEBAQEBAQEBAQEBAQEBAQEB' +
+    'AQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEB' +
+    '/8AAEQgAAQACAwEiAAIRAQMRAf/EAB8AAAEFAQEBAQEBAAAAAAAAAAABAgMEBQYH' +
+    'CAkKC//EALUQAAIBAwMCBAMFBQQEAAABfQECAwAEEQUSITFBBhNRYQcicRQygZGh' +
+    'CCNCscEVUtHwJDNicoIJChYXGBkaJSYnKCkqNDU2Nzg5OkNERUZHSElKU1RVVldY' +
+    'WVpjZGVmZ2hpanN0dXZ3eHl6g4SFhoeIiYqSk5SVlpeYmZqio6Slpqeoqaqys7S1' +
+    'tre4ubrCw8TFxsfIycrS09TV1tfY2drh4uPk5ebn6Onq8fLz9PX29/j5+v/EAB8B' +
+    'AAMBAQEBAQEBAQEAAAAAAAABAgMEBQYHCAkKC//EALURAAIBAgQEAwQHBQQEAAEC' +
+    'dwABAgMRBAUhMQYSQVEHYXETIjKBCBRCkaGxwQkjM1LwFWJy0QoWJDThJfEXGBka' +
+    'JicoKSo1Njc4OTpDREVGR0hJSlNUVVZXWFlaY2RlZmdoaWpzdHV2d3h5eoKDhIWG' +
+    'h4iJipKTlJWWl5iZmqKjpKWmp6ipqrKztLW2t7i5usLDxMXGx8jJytLT1NXW19jZ' +
+    '2uLj5OXm5+jp6vLz9PX29/j5+v/aAAwDAQACEQMRAD8A/v4ooooA/9k=';
 const imageUrlJPEG = 'data:image/jpeg;base64,' + b64DataJPEG;
 const imageBlob = new Blob([imageUrlJPEG]);
 
 // Callback style
-loadImage(imageUrlJPEG, (image: Event | HTMLCanvasElement | HTMLImageElement, data?: loadImage.MetaData): void => {
-  const canvas = image as HTMLCanvasElement;
-  console.log(data);
-  canvas.toBlob((blob: Blob | null): void => {
-    const url = canvas.toDataURL("image/png");
-    console.log(url);
-  });
-}, { canvas: true, orientation: true, maxWidth: 100, maxHeight: 100, crop: true });
+loadImage(
+    imageUrlJPEG,
+    (image: Event | HTMLCanvasElement | HTMLImageElement, data?: loadImage.MetaData): void => {
+        const canvas = image as HTMLCanvasElement;
+        console.log(data);
+        canvas.toBlob((blob: Blob | null): void => {
+            const url = canvas.toDataURL('image/png');
+            console.log(url);
+        });
+    },
+    { canvas: true, orientation: true, maxWidth: 100, maxHeight: 100, crop: true },
+);
 
 // Promise style
-loadImage(imageUrlJPEG, { canvas: true, orientation: true, maxWidth: 100, maxHeight: 100, crop: true }).then((data) => {
-  const canvas = (<any> data.image) as HTMLCanvasElement;
-  console.log(data);
-  canvas.toBlob((blob: Blob | null): void => {
-    const url = canvas.toDataURL("image/png");
-    console.log(url);
-  });
+loadImage(imageUrlJPEG, { canvas: true, orientation: true, maxWidth: 100, maxHeight: 100, crop: true }).then(data => {
+    const canvas = (<any>data.image) as HTMLCanvasElement;
```

Comment:
@StampixSMO I'm not sure either I can't get it to fail locally but it's obviously having issue with the angle bracket type-cast (which had a space after it before but doesn't with your changes).

You could try changing it to `as` and see if it still complains:

```suggestion
    const canvas = data.image as any as HTMLCanvasElement;
```

Either that or just add the space back so it is the way it was before (even though local prettier complains then)

```suggestion
    const canvas = (<any> data.image) as HTMLCanvasElement;
```

Reply:
Your own prettier seems to have changed things pretty aggressively. It makes it hard to see what you have actually changed across the files.

Reply:
Oops, I was running the given prettier command in the "Common Mistakes" file so was assuming it would use a shared configuration. Need me to revert those?